les/clientpool.go

```diff
@@ -611,6 +741,10 @@ func (db *nodeDB) getOrNewPB(id enode.ID) posBalance {
 }
 
 func (db *nodeDB) setPB(id enode.ID, b posBalance) {
+	if b.value == 0 && len(b.meta) == 0 {
```

Comment:
Please use `nodeDB.delPB` directly.

Reply:
As I mentioned above, I want to keep entries with non-empty meta tags.